src/material.js

```diff
@@ -99,6 +99,12 @@ export class Material {
         return this.prop('isNonPeriodic', false, true);
     }
 
+    /**
+     * @summary Sets the value of isNonPeriodic based on Boolean value passed as an argument.
+     * @param bool
+     */
+    updateIsNonPeriodic(bool) { this.prop('isNonPeriodic', bool);}
```

Comment:
This doesn't update anything actually.

See like 121 of this file for how the update should be done

Reply:
Yes, sorry this got updated in a later PR. I will update here as well.